### PR TITLE
Govern test

### LINKED ISSUE #14: Test Govern.sol

### PR DESCRIPTION
**Pull request by nivida**
closes #14

### CODE REVIEW

File: packages/govern-core/test/govern.test.ts [hunk 1]
```diff
@@ -39,6 +44,45 @@ describe('Govern', function () {
...
+
+
+    await expect(signers[0].sendTransaction(
+      {
+        to: govern.address,
+        data: callbackSig
+      }
+      )
+    )
+    .to.emit(govern, EVENTS.RECEIVED_CALLBACK)
```
Comment (kanaagawa) [inline packages/govern-core/test/govern.test.ts:82]: Not sure if possible with a low level return, but maybe we can also assert the right `magicNumber` got returned.

**Review by 0xGabi — APPROVED** [review]
LGTM 🙌

File: packages/govern-core/test/govern.test.ts (hunk 1 above)
Comment (nivida) [reply packages/govern-core/test/govern.test.ts:82]: I think we can add this test case in the ``AdaptativeERC165`` test where it actually belongs to or we would duplicate the test cases too much. I was just not able to mock the extended contract.
